config/manager/manager.yaml

```diff
@@ -36,6 +36,10 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: RESTIC_PV_HOSTPATH
```

Comment:
Since there is only one function, can we add just one env var?

Reply:
I was thinking maybe we keep both and remove the restic one once we deprecate it.

Reply:
@shubham-pampattiwar well not when we deprecate it but when we remove it.